src/portal/package.json

```diff
@@ -32,18 +32,26 @@
         "@clr/angular": "^0.12.10",
         "@clr/icons": "^0.12.0",
         "@clr/ui": "^0.12.0",
+        "@fortawesome/fontawesome-free": "^5.1.0-4",
+        "@ng-bootstrap/ng-bootstrap": "^2.0.0",
         "@ngx-translate/core": "^10.0.2",
         "@ngx-translate/http-loader": "^3.0.1",
         "@types/jquery": "^2.0.41",
         "@webcomponents/custom-elements": "^1.1.3",
+        "bootstrap": "^4.1.1",
+        "buffer": "^5.2.1",
         "core-js": "^2.5.4",
         "intl": "^1.2.5",
+        "jquery": "^3.3.1",
```

Comment:
jquery is already included in types/jquery, I think we can remove it from `package.json`

Reply:
The version is different from what swagger-ui depends.